pkgs/development/ocaml-modules/opam-format/default.nix

```diff
@@ -0,0 +1,22 @@
+{ lib, buildDunePackage, unzip, opam-core, opam-file-format }:
+
+buildDunePackage rec {
+  pname = "opam-format";
+
+  inherit (opam-core) src version;
+
+  minimumOCamlVersion = "4.02.3";
+
+  # get rid of check for curl at configure time
+  # opam-format does not call curl at run time
+  configureFlags = [ "--disable-checks" ];
+
+  nativeBuildInputs = [ unzip ];
+  propagatedBuildInputs = [ opam-core opam-file-format ];
+
+  meta = with lib; {
+    inherit (opam-core.meta) homepage;
```

Comment:
License?

Reply:
Done in ca8a57027b911fe8b2f934f8b8529c97092f39ee.